netlify.toml

```diff
@@ -15,3 +15,9 @@
 
 [context.branch-deploy]
     command = "hugo -b $DEPLOY_PRIME_URL"
+
+[[redirects]]
+    from = "/*"
+    to = "/maintenance"
+    status = 200
+    force = false
```

Comment:
If you set `force = false`, it will not redirect whenever there's a "real" page existing, i.e `/`, `/services`, `/blog`, etc. I think what you want is `force = true`.